command/src/channel.rs

```diff
@@ -86,8 +89,18 @@ impl<Tx: Debug + Serialize, Rx: Debug + DeserializeOwned> Channel<Tx, Rx> {
         self.blocking = !nonblocking;
     }
 
-    pub fn set_blocking(&mut self, blocking: bool) {
-        self.set_nonblocking(!blocking)
+    /// set the channel to be blocking
+    pub fn block(&mut self) {
+        self.set_nonblocking(false)
+    }
+
+    /// set the channel to be nonblocking
+    pub fn unblock(&mut self) {
```

Comment:
I think it will be better if the name better match the inner method like `nonblocking`